vscode/src/edit/input/get-items/range.ts

```diff
@@ -22,6 +27,11 @@ const getDefaultRangeItems = (
         range: new vscode.Range(initialCursorPosition, initialCursorPosition),
     }
 
+    const fullItem = {
+        ...FULL_RANGE_ITEM,
+        range: new vscode.Range(0, 0, document.lineCount + 1, 0),
+    }
```

Comment:
```suggestion
    const fullItem = {
        ...FULL_RANGE_ITEM,
        range: new vscode.Range(0, 0, document.lineCount - 1, 0),
    }
```

Otherwise will throw an error I think

Reply:
Thanks this was really easy to reproduce the issue. Apologies, you shouldn't have to do more testing than me :) Fix pushed.